crates/fluvio-controlplane-metadata/src/smartmodule/package.rs

```diff
@@ -97,6 +106,18 @@ pub enum SmartModuleKeyError {
     InvalidVersion { version: String, error: String },
 }
 
+#[derive(Debug, Default, Clone, PartialEq, Eq, Encoder, Decoder)]
+#[cfg_attr(
+    feature = "use_serde",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(rename_all = "lowercase")
+)]
+pub enum SmartModuleVisibility {
+    #[default]
```

Comment:
may need customer encoder/decoder to compatible which assumes public as default

Reply:
Since there aren't any customer public packages yet other than infinyon and our our own demos - it seems better to default to private as requested in the Product RFD. All we need to do is re-publish the public infinyon packages.

Reply:
K